stable/rabbitmq/values-production.yaml

```diff
@@ -226,3 +227,16 @@ metrics:
   annotations:
     prometheus.io/scrape: "true"
     prometheus.io/port: "9090"
+
+##
+## Init containers parameters:
+## volumePermissions: Change the owner of the persist volume mountpoint to RunAsUser:fsGroup
+##
+volumePermissions:
+  enabled: true
+  image:
+    registry: docker.io
+    repository: bitnami/minideb
+    tag: latest
+    pullPolicy: IfNotPresent
+  resources: {}
```

Comment:
This parameter `volumePermissions.resources` is not documented in the README.md

Reply:
Will fix this